quic/s2n-quic-transport/src/space/mod.rs

```diff
@@ -441,6 +441,10 @@ impl<Config: endpoint::Config> PacketSpaceManager<Config> {
     pub fn retry_cid(&self) -> Option<&PeerId> {
         self.retry_cid.as_deref()
     }
+
+    pub fn has_ack_interest(&self) -> bool {
```

Comment:
Thinking this could prob be a Provider like transmission and timers but wasnt sure if that was overkill.

But talking about it more, it would prob be a good idea once we start to implement some of the other delayed ack mechanisms.